
<launch>
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find vio_system)/config/rviz/vio_rviz_config.rviz" />



    <node name="vio_estimator" pkg="vio_system" type="vio_system_node" output="screen" args="$(find vio_system)/config/tum/tum_mono_config.yaml" />

    
</launch>
